src/ObjectUtils/LinuxMapTest.cpp

```diff
@@ -206,4 +206,216 @@ TEST(LinuxMap, ParseMaps) {
   }
 }
 
+TEST(LinuxMap, ParseMapsWithPeTextMappedAnonymouslyAtExpectedOffset) {
+  const std::filesystem::path test_path = orbit_test::GetTestdataDir();
+  const std::filesystem::path libtest_path = test_path / "libtest.dll";
+
+  const std::string data{
+      absl::StrFormat("100000-101000 r--p 00000000 01:02 42    %s\n"
+                      "101000-103000 r-xp 00000000 00:00 0\n",
+                      libtest_path)};
+  const auto result = ParseMaps(data);
+  ASSERT_THAT(result, HasNoError());
+  ASSERT_EQ(result.value().size(), 1);
+
+  const orbit_grpc_protos::ModuleInfo& libtest_module_info = result.value()[0];
+  EXPECT_EQ(libtest_module_info.name(), "libtest.dll");
+  EXPECT_EQ(libtest_module_info.file_path(), libtest_path);
+  EXPECT_EQ(libtest_module_info.file_size(), 96441);
+  EXPECT_EQ(libtest_module_info.address_start(), 0x101000);
+  EXPECT_EQ(libtest_module_info.address_end(), 0x103000);
+  EXPECT_EQ(libtest_module_info.build_id(), "");
+  EXPECT_EQ(libtest_module_info.load_bias(), 0x62640000);
```

Comment:
Our of curiosity: How does the load bias computation work here? Where do I get the information from the maps entry/the test file?